### ISSUE TITLE
Med picker is hard to use

### ISSUE THREAD

**Issue by jmandel** [cds-hooks/sandbox#125]
I've seen users get stuck without realizing there was still more data entry to do before a med is fully "selected":

![image](https://user-images.githubusercontent.com/313089/70922193-4d20eb00-1feb-11ea-852c-c488c2de8563.png)

In this screenshot, it's not obvious that you have to click one more time to finish selecting a med. (You have to click to choose a drug; then a dose form; then a semantic clinical drug -- which means three clicks, and the UX doesn't make this progression / requirement clear.)

**Comment by barabo** [comment]
I have to agree - the first time I tried to reproduce this issue I was confused by requiring the third click to finalize the selection.

I would offer two suggestions to improve the UI which should both be easy to implement.

- Disable (or hide) the 'Number' and 'Frequency' controls until a drug selection has been completed.
- Include a hint phrase in the place where the selected drug will eventually go.  I.E. `Select a Medication and dose above...`

![image](https://user-images.githubusercontent.com/4342684/71022350-e1f81700-20c5-11ea-890c-ae8f81bc835a.png)
